mdms/mdms-java/src/main/java/de/hpi/isg/mdms/java/apps/ForeignKeyClassifier.java

```diff
@@ -379,7 +407,7 @@ public String explain(DependencyPrettyPrinter prettyPrinter) {
 
         @Parameter(names = {"--distinct-values"},
                 description = "ID of the constraint collection that contains the distinct value counts",
-                required = true)
+                required = false)
         public int dvcCollectionId;
```

Comment:
This parameter is probably not longer needed, is it? So we could also remove it...